### ISSUE TITLE
Springs Enchantment gives error.

### ISSUE THREAD

**Issue by Klawdek** [badbones69/Crazy-Enchantments#586]
**Describe the bug**

When I use boots with Springs enchantment I get the following error.  And the springs do not work.  There is no jump height increase.

https://pastebin.com/FMpZesJT

I do not get this problem on my local server.  Yet it is happening on my public server.  On the public server I have a number of enchantments disabled. I have had problems in the past where disabling one enchantment causes another enchantment to malfunction. So I took the enchantments.yml from my private server which has no enchantments disabled and put it on my public server and now it works.  The problem is I need to disable certain enchantments because they are PvP and a couple because they are expensive dupes of vanilla enchantments and at least one because it does not work at all.

After the first time trying to put on the boots.  It will give the following shorter error every time the boots are put on 
https://pastebin.com/Wdsh6SfY

I have tried moving the enchantments file from my public server to my private and the errors follow along.  It also affects other enchantments when I tried to put on a chest plate with Hulk and BurnShield I get the error.

It seems disabling enchantments causes other enchantments to malfunction.

Here is a copy of my enchantments.yml
https://pastebin.com/a9YRtvKC

**Versions (please complete the following information):**
 - Minecraft Server Version: [e.g. 1.8.8] git-Paper-564 (MC: 1.16.5) (Implementing API version 1.16.5-R0.1-SNAPSHOT)
 - Plugin Version: [e.g. 1.7.9.4] 1.8-Dev-Build-v8.1-SNAPSHOT-Build#168

*Do **NOT** say Latest use `/version crazyenchantments` in the console.

**Pastebin any errors in console**
Link: https://pastebin.com/

**Issue**
Describe the issue you are having with the plugin.

**Screenshots**

**Comment by Klawdek** [comment]
I did some more experimenting.  I took the enchantments.yml file from my local server where CE was installed fresh.  I then turned off all of the stuff that is turned off on my public server and it worked.  The only difference in the two files I noticed was the new enchantments.yml file has an entry for beekeeper and the old one does not.  There could be other differences that I did not see.  It seems I should have done a fresh install on my public server and let the enchantments.yml get generated new.
